misk/src/main/kotlin/misk/client/HttpClientFactory.kt

```diff
@@ -8,11 +8,25 @@ import okhttp3.OkHttpClient
 import okhttp3.Protocol
 import java.io.File
 import java.net.Proxy
+import java.time.Duration
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 import javax.inject.Singleton
 import javax.net.ssl.X509TrustManager
 
+
+private object Defaults {
+  /*
```

Comment:
Should these be `/**`?

Reply:
Not sure. IMHO `/**` is meant to be a part of Javadoc, and describe the meaning of the field and it's usage. In this case, the comment is more about meta-information, it's almost a TODO or an explanatory note, as in "If by any chance those fields become public in ConnectionPool - remove them from here".